swan-lake/resources/featured-scenarios/create-your-first-connector-with-ballerina.md

```diff
@@ -0,0 +1,176 @@
+2. An OpenAPI specification of the API for which you’re building the connector, along with the relevant API credentials where required (e.g., Twitter Developer account keys).
+3. A GitHub account, and Git installed locally.
+4. Visual Studio Code with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed.
+
+## Step 1: Set the project structure
+
+> **Important**:If you'd like to contribute your project as an official Ballerina connector, you can skip the steps below and, reach out to the Ballerina team to create a new repository (under [`ballerina-platform`](https://github.com/ballerina-platform/) organization) and to set up the initial project structure. 
+
+1. Create a new GitHub repository to host your connector. Ballerina official connector repositories follow the naming pattern: `module-ballerinax-<connector-name>` (e.g., module-ballerinax-twitter). But for other connectors, you can choose a name that best represents your connector.
+
+2. Clone your newly created repository to your local machine:
+
+   ```
+   git clone https://github.com/<your-username>/<connector-repo-name>.git
+   cd <connector-repo-name>
+   ```
+
+3. Visit the [Ballerina generated connector template on GitHub](https://github.com/ballerina-platform/ballerina-library/tree/main/library-templates/generated-connector-template/files) and copy the entire project structure and content to your local repository folder, making sure to include all files and directories.
+
+4. Your local project structure should now look similar to this:
+
+   ```
+   module-ballerinax-myconnector/
+   ├── .github/
+   ├── ballerina/
+   |   ├── tests/
+   │   ├── Ballerina.toml
+   │   ├── Module.md
+   │   ├── Package.md
+   │   ├── build.gradle
+   │   └── client.bal
+   ├── build-config/
+   ├── docs/
+   │   └── spec/
+   │       └── sanitations.md
+   ├── examples/
+   │   ├── README.md
+   │   ├── build.gradle
+   │   └── build.sh
+   ├── gradle/
+   ├── .gitignore
+   ├── LICENSE
+   ├── README.md
+   ├── build.gradle
+   ├── gradle.properties
+   ├── gradlew
+   ├── gradlew.bat
+   └── settings.gradle
+   ```
+
+> **Tip**: The template includes placeholders for various fields. To update them with your connector-specific metadata, use the provided [Ballerina script](https://github.com/ballerina-platform/ballerina-library/blob/main/library-templates/generated-connector-template/scripts/replace_placeholders.bal).
+
+Detailed information on the Ballerina connector structure can be found in the [Ballerina module contribution guide](https://github.com/ballerina-platform/ballerina-library/blob/main/docs/adding-a-new-ballerina-module.md#directory-structure).
+
+## Step 2: Prepare the OpenAPI Specification
+
+1. Find the OpenAPI specification for the API you want to create a connector for. This is usually available in the API documentation.
```

Comment:
We don't need `for` at the end?

Reply:
It seems like having `for` at the end is more grammatically correct in this case (verified with GPT too)